lib/trento_web/controllers/v1/profile_controller.ex

```diff
@@ -41,4 +41,61 @@ defmodule TrentoWeb.V1.ProfileController do
       render(conn, "profile.json", user: updated_user)
     end
   end
+
+  operation :reset_totp,
+    summary: "Reset the TOTP configuration for the user",
+    tags: ["Platform"],
+    responses: [
+      forbidden: Schema.Forbidden.response()
+    ]
+
+  def reset_totp(conn, _) do
+    %User{} = user = Pow.Plug.current_user(conn)
+
+    with {:ok, %User{}} <- Users.reset_totp(user) do
+      send_resp(conn, :no_content, "")
+    end
+  end
+
+  operation :get_totp_enrollment_data,
+    summary: "Get TOTP enrollment data",
+    tags: ["Platform"],
+    responses: [
+      ok:
+        {"UserTOTPEnrollmentPayload", "application/json", Schema.User.UserTOTPEnrollmentPayload},
+      unprocessable_entity: Schema.UnprocessableEntity.response(),
+      forbidden: Schema.Forbidden.response()
+    ]
+
+  def get_totp_enrollment_data(conn, _) do
+    %User{} = user = Pow.Plug.current_user(conn)
+
+    with {:ok, enrollment_payload} <- Users.initiate_totp_enrollment(user) do
+      render(conn, "totp_enrollment_data.json", enrollment_payload: enrollment_payload)
+    end
+  end
+
+  operation :confirm_totp_enrollment,
+    summary: "Confirm TOTP enrollment procedure",
+    tags: ["Platform"],
+    request_body:
+      {"UserTOTPEnrollmentConfirmRequest", "application/json",
+       Schema.User.UserTOTPEnrollmentConfirmRequest},
+    responses: [
+      ok:
+        {"TOTP Enrollment completed", "application/json",
+         Schema.User.UserTOTPEnrollmentConfirmPayload},
+      unprocessable_entity: Schema.UnprocessableEntity.response(),
+      forbidden: Schema.Forbidden.response()
+    ]
+
+  def confirm_totp_enrollment(%{body_params: body_params} = conn, _) do
+    %User{} = user = Pow.Plug.current_user(conn)
+    totp = Map.get(body_params, :totp)
```

Comment:
Again, I'm using `totp_code` in the other PR. Let's use the same term.
I will adapt my PR once we have it decided

Reply:
totp_code also here and in the schemas!